docs/context/compaction.md

```diff
@@ -0,0 +1,63 @@
+# Compress workflow context for performance
```

Comment:
was wondering if you plan to cover this: https://github.com/google/adk-python/blob/main/src/google/adk/apps/base_events_summarizer.py

Reply:
users can customize their summary prompt through: https://github.com/google/adk-python/blob/main/src/google/adk/apps/llm_event_summarizer.py#L60